stacker_blueprints/conveyor/__init__.py

```diff
@@ -0,0 +1,482 @@
+                    '/root/.docker/config.json': cf.InitFile(
+                        content=self.local_parameters['DockerConfig'],
+                        mode='000600',
+                        owner='root',
+                        group='root',
+                    ),
+                    '/var/run/conveyor/.docker/config.json': cf.InitFile(
+                        content=self.local_parameters['DockerConfig'],
+                        mode='000600',
+                        owner='root',
+                        group='root',
+                    ),
+                    '/var/run/conveyor/.ssh/id_rsa': cf.InitFile(
+                        content=self.local_parameters['SshIdRsa'],
+                        mode='000600',
+                        owner='root',
+                        group='root',
+                    ),
+                    '/var/run/conveyor/.ssh/id_rsa.pub': cf.InitFile(
+                        content=self.local_parameters['SshIdRsaPub'],
+                        mode='000600',
+                        owner='root',
+                        group='root',
+                    ),
+                }),
+            ),
+        })
+
+    def create_conditions(self):
+        t = self.template
+        t.add_condition(
+            'NoSSL',
+            Equals(Ref('SSLCertificateName'), ''),
+        )
+        t.add_condition(
+            'UseDNS',
+            Not(Equals(Ref('ExternalDomain'), '')),
+        )
+
+    def create_security_groups(self):
+        t = self.template
+
+        t.add_resource(
+            ec2.SecurityGroup(
+                CLUSTER_SG_NAME,
+                GroupDescription=CLUSTER_SG_NAME,
+                VpcId=Ref('VpcId'),
+            ),
+        )
+        t.add_resource(
+            ec2.SecurityGroup(
+                ELB_SG_NAME,
+                GroupDescription=ELB_SG_NAME,
+                VpcId=Ref('VpcId'),
+            ),
+        )
+        t.add_resource(
+            ec2.SecurityGroupIngress(
+                'ConveyorELBFromTrusted',
+                IpProtocol='tcp',
+                FromPort=If('NoSSL', '80', '443'),
+                ToPort=If('NoSSL', '80', '443'),
+                CidrIp=Ref('TrustedNetwork'),
+                GroupId=Ref(ELB_SG_NAME),
+            ),
+        )
+        t.add_resource(
+            ec2.SecurityGroupIngress(
+                'ConveyorELBToConveyorCluster',
+                IpProtocol='tcp',
+                FromPort='8080',
+                ToPort='8080',
+                SourceSecurityGroupId=Ref(ELB_SG_NAME),
+                GroupId=Ref(CLUSTER_SG_NAME),
+            ),
+        )
+
+    def create_load_balancer(self):
+        t = self.template
+        t.add_resource(
+            elb.LoadBalancer(
+                LOAD_BALANCER,
+                HealthCheck=elb.HealthCheck(
+                    Target='TCP:8080',
+                    HealthyThreshold=3,
+                    UnhealthyThreshold=3,
+                    Interval=30,
```

Comment:
This seems like a crazy long interval - probably a leftover from a workaround after dealing w/ ECS and it's poor behavior with failed containers? @ejholmes